noir/aztec_macros/src/lib.rs

```diff
@@ -576,6 +774,187 @@ fn transform_events(
     Ok(())
 }
 
+/// Obtains the serialized length of a type that implements the Serialize trait.
+fn get_serialized_length(
+    traits: &Vec<TraitId>,
```

Comment:
We can use a slice instead of a specific reference to a vec &[TraitId]

Reply:
Done!